toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -234,9 +248,11 @@ func parsePackListFile(packListFile string) (packList []string, err error) {
 func createAllSRPMsWrapper(specsDir, distTag, buildDir, outDir, workerTar string, workers int, nestedSourcesDir, repackAll, runCheck bool, packList []string, templateSrcConfig sourceRetrievalConfiguration) (err error) {
 	var chroot *safechroot.Chroot
 	originalOutDir := outDir
+	originalCacheDir := templateSrcConfig.localSourceCacheDir
```

Comment:
Small variable naming suggestion: let's keep the word "source" in there so it's always clear what cache we're talking about. So: `originalCacheDir` -> `originalSourceCacheDir` and `cacheInChroot` -> `sourceCacheInChroot`.